frontend/src/app/components/tableWithFilter/table.component.html

```diff
@@ -0,0 +1,79 @@
+<p-table #datatable [value]="data" dataKey="id" [rows]="9" [showCurrentPageReport]="true"
```

Comment:
Why does the table contain filtering? The filtering is another issue which should be done separately.

Reply:
des is aber wohl sehr kritisch, wenn i de Tabelle mach, hau i glei alles rein sry :D lösch i halt alles und paste des in an neuen branch :D